src/ophys_etl/transforms/roi_transforms.py

```diff
@@ -127,3 +129,149 @@ def crop_roi_mask(roi_mask: coo_matrix) -> coo_matrix:
     cropped_mask = roi_mask.tocsr()[min_row:max_row, min_col:max_col]
 
     return cropped_mask.tocoo()
+
+
+def coo_rois_to_old(coo_masks: List[coo_matrix],
+                    max_correction_vals: motion_border,
+                    movie_shape: Tuple[int, int]) -> List[Dict[str, Any]]:
+    """
+    Converts coo formatted ROIs to old format from previous Ophys Segmentation
+    implementation. This is a required transformation as the next workflow
+    step, Extract Traces, expects ROIs in this format and to reduce work load
+    this will not be changed.
+    Parameters
+    ----------
+    coo_masks: List[coo_matrix]
+        A list of scipy coo_matrices representing ROI masks, each element of
+        list is a unique ROI.
+    max_correction_vals: motion_border
+        The max motion correction values identified in the motion correction
+        step of ophys segmentation pipeline
+    movie_shape: Tuple[int, int]
+        The frame shape of the movie from which ROIs were extracted in order
+        of: (height, width).
+
+    Returns
+    -------
+    List[Dict[str, Any]]
+        A list of dictionaries representing the ROIs in old segmentation
+        format, the data contained inside each one is as follows:
+        {
+            id: int
+            x: int
+            y: int
+            width: int
+            height: int
+            valid_roi: Bool
+            mask_matrix: List[List[Bool]]
+            max_correction_up: int
+            max_correction_down: int
+            max_correction_left: int
+            max_correction_right: int
+            mask_image_plane: int
+            exclusion_labels: List[int] codes are defined in 2019 Ophys Docs
+        }
+        For details about specific values see design document for 2020 Ophys
+        Segmentation Refactor and Update
+        """
+    old_rois = []
+    for temp_id, coo_mask in enumerate(coo_masks):
+        old_roi = _coo_mask_to_old_format(coo_mask)
+        old_roi['id'] = temp_id  # popped off writing to LIMs
+        old_roi['cell_specimen_id'] = temp_id  # updated post nway cellmatching
+        old_roi['max_correction_up'] = int(max_correction_vals.up)
```

Comment:
Why are these casted to int? I realize they need to be in order to check for border exclusions, but LIMS stores these as double precision. (example: SELECT * FROM cell_rois LIMIT 10)

Reply:
They shouldn't be, it's an artifact from when I was debugging. Good catch, the casts have been removed.

Reply:
Why was the fact that the outputs were casted incorrectly not caught in tests?

Reply:
The motion border for this particular test was all ints it is now in floats.